src/hash_set/managed.zig

```diff
@@ -51,6 +65,17 @@ pub fn HashSetManaged(comptime E: type) type {
             return .{
                 .allocator = allocator,
                 .map = Map.init(),
+                .context = if (Context == void) {} else undefined,
```

Comment:
For line 68 and 69: can you explain why if the Context is void `{}` is getting set for both fields? I'm not grokking that syntax for some reason.